test/lib/seeds_test.rb

```diff
@@ -50,4 +78,18 @@ def test_export_all_with_no_site
     end
   end
 
+  def test_export_single_class
+    host_path = File.join(ComfortableMexicanSofa.config.seeds_path, "test-site")
+    ComfortableMexicanSofa::Seeds::Exporter.new("default-site", "test-site").export!(["Layout"])
+    assert(File.exist?(File.join(host_path, "layouts")))
+    FileUtils.rm_rf(host_path)
+  end
+
+  def test_export_multiple_classes
+    host_path = File.join(ComfortableMexicanSofa.config.seeds_path, "test-site")
+    ComfortableMexicanSofa::Seeds::Exporter.new("default-site", "test-site").export!(%w[Layout Snippet])
+    assert(%w[layouts snippets].all? {|klass| File.exist?(File.join(host_path, klass))})
+    FileUtils.rm_rf(host_path)
```

Comment:
Good tests. Just realized that `FileUtils.rm_rf(host_path)` should be under `ensure` so cleanup happens even if the test fails. Feel free to adjust other tests.